lib/pact/jwt.rb

```diff
@@ -0,0 +1,54 @@
+require 'pact/symbolize_keys'
+module Pact
+
+  # Specifies that the actual object should be considered a match if
+  # it includes the same keys, and the values of the keys are of the same class.
+
+  class Jwt
```

Comment:
I don't like Jwt very much, to be honest. I started with JWT, and changed it, to avoid confusion with JWT the gem itself.